libbeat/processors/add_process_metadata/config.go

```diff
@@ -75,6 +84,8 @@ func defaultConfig() config {
 		IgnoreMissing:    true,
 		OverwriteKeys:    false,
 		RestrictedFields: false,
+		HostPath:         "/",
+		CgroupPrefixes:   []string{"/kubepods", "/docker"},
```

Comment:
@andrewkroh What's your opinion on adding a default for `MatchPIDs:        []string{"process.pid", "process.ppid"}`?

Reply:
SGTM. Those are the standard ECS fields for this information.

Reply:
And if you wanted to be extra thorough also include `process.parent.pid` (which is same as `process.ppid`) and `process.parent.ppid` (grandparent).

Reply:
@BrookHF would you mind adding these as defaults? I can do that in a follow up PR too

Reply:
No problem, I added those as default.